lib/traject/config/sdr_config.rb

```diff
@@ -331,6 +331,12 @@ def mods_display(method, *args, default: nil)
 
 to_field 'summary_display', mods_display(:abstract)
 
+to_field 'iiif_manifest_url_ssim' do |record, accumulator|
+  if %w[image manuscript map book].include?(record.dor_content_type)
```

Comment:
The logic in purl for whether an object has IIIF material is a little different:

https://github.com/sul-dlss/purl/blob/master/app/models/iiif_presentation_manifest.rb#L21-L26

Do these need to be reconciled at all?

Reply:
I guess is the outstanding case here where content is a book, but doesn't have pages/images?

Reply:
I think we should update this logic in PURL to be consistent with this logic.